src/components/PurchaseOrder/PODetails/PODetailsView.js

```diff
@@ -26,92 +27,107 @@ class PODetailsView extends Component {
     const addressBillTo = get(addresses.find(el => el.id === get(order, 'billTo', '')), 'address', '');
     const addressShipTo = get(addresses.find(el => el.id === get(order, 'shipTo', '')), 'address', '');
 
+    const metadata = get(order, 'metadata');
```

Comment:
I think you need to use something like this
`{initialValues.metadata && <this.cViewMetaData metadata={initialValues.metadata} />}`
https://github.com/folio-org/ui-invoice/pull/17/files#diff-08aec453035a9e78f35e04a26b8a7280R115

Reply:
and you need `ViewMetaData`

Reply:
and you need `ViewMetaData`

Reply:
Done